### PR TITLE
v1.3.0

### PR DESCRIPTION
**Pull request by moshe-blox**
## Changes
- Deneb spec alignment (#1282) 
- Align spec with 375a2766d19e (#1252) 
- Upgrade ssv-spec to v0.3.6 (#1299)
- fix: (API) missing nil check in byIndices (#1283)
- fix: (SlotTicker) potential double tick (#1244) 
- fix: don't start liquidated validators after metadata fetch (#1273)
- Don't use BeaconState in Validators call (#1301)
- Initial E2E implementation (#1281) 
- removed any reference for operator id in node metadata (#1275) 
- test: ValidatorController unit tests (#1124) 
- Missing Slashing Tests (#1241)
- E2E align to latest (#1302)
- fix: e2e - clean up before running & stop run if any command exited (#1295)

### CODE REVIEW

**Review by lior-blox — APPROVED** [review]

**Review by nkryuchkov — APPROVED** [review]

File: network/records/node_info_test.go [hunk 1]
```diff
@@ -32,21 +31,25 @@ func TestNodeInfo_Seal_Consume(t *testing.T) {
 }
 
 func TestNodeInfo_Marshal_Unmarshal(t *testing.T) {
-	ni := &NodeInfo{
+	oldSerializedData := []byte(`{"Entries":["", "testnet", "{\"NodeVersion\":\"v0.1.12\",\"ExecutionNode\":\"geth/x\",\"ConsensusNode\":\"prysm/x\",\"OperatorsID\":\"xxx\"}"]}`)
```
Comment (moshe-blox) [inline network/records/node_info_test.go:34]: typo: OperatorsID (should be OperatorID)

**Review by lior-blox — APPROVED** [review]

File: e2e/go.mod [hunk 2]
```diff
@@ -0,0 +1,115 @@
+module github.com/bloxapp/ssv/e2e
+
+go 1.20
+
+require (
+	github.com/attestantio/go-eth2-client v0.16.3
+	github.com/bloxapp/ssv v1.2.1-0.20231204141016-a6afbe837a26
```
Comment (olegshmuelov) [inline e2e/go.mod:7]: should the e2e be updated as well?

Comment (moshe-blox) [reply e2e/go.mod:7]: Since it doesn't do proposals yet then there's a chance that it doesn't need to, right? @y0sher

Comment (y0sher) [reply e2e/go.mod:7]: gonna try running it, not sure since go-eth2-client has been updated not just in proposals.

**Review by olegshmuelov — APPROVED** [review]

**Review by y0sher — APPROVED** [review]

**Review by lior-blox — APPROVED** [review]

**Pull request merged**